### PR TITLE
add for loop

### PR DESCRIPTION
**Pull request by plskz**
I added a for loop please check it

### CODE REVIEW

File: hello.cpp [hunk 1]
```diff
@@ -6,5 +6,11 @@ int main() {
     cout << "ABCD\n";
     cout << "EFGH";
 
+// make a comment that i forgot the n
+
```
Comment (AvinashDhillor) [inline hello.cpp:10]: Please add n in code.

**Review by AvinashDhillor — CHANGES_REQUESTED** [review]

File: hello.cpp (hunk 1 above)
Comment (plskz) [reply hello.cpp:10]: ok im adding it now

Comment (plskz) [reply hello.cpp:10]: and removing the comments

**Comment by plskz** [comment]
test

**Comment by AvinashDhillor** [comment]
checking

**Pull request merged**